kit/inputs/src/UIKeyTextView.tsx

```diff
@@ -93,6 +103,16 @@ export function useKeyTextView(
         };
     }, [inputHasValue, hasInvalidChars, hasProperLength, isFocused]);
 
+    React.useEffect(() => {
```

Comment:
What do you think to split to two different `useEffect`s, I'm not quite understood why you want `onError` to be called, when only `success` is changed

Reply:
https://github.com/tonlabs/UIKit/pull/1476/commits/9a0447c5cc35417b0ecb3622a1aaebf571c8e06b